junit-platform-engine/src/main/java/org/junit/platform/engine/Namespace.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2015-2025 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.engine;
+
+import static org.apiguardian.api.API.Status.STABLE;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.apiguardian.api.API;
+import org.junit.platform.commons.util.Preconditions;
+
+@API(status = STABLE, since = "5.13")
```

Comment:
We usually introduce new APIs as `EXPERIMENTAL` in their initial release

```suggestion
@API(status = EXPERIMENTAL, since = "5.13")
```